OpenChange/RTFHandler.m

```diff
@@ -332,6 +343,23 @@ - (RTFFontInfo *) fontInfoAtIndex: (unsigned int) theIndex
   return NSMapGet(fontInfos, key);
 }
 
+- (NSString *) description
+{
+  NSMutableString *description;
+  NSEnumerator *enumerator;
+  RTFFontInfo *fontInfo;
+
+  description = [NSMutableString stringWithFormat: @"Number of fonts:%u\n", [fontInfos count]];
```

Comment:
When one of this two pulls is merged, I will rebase the other and fix the conflicts.